acceptance/image/image.go

```diff
@@ -706,17 +703,14 @@ func JSONImageSignaturesFrom(ctx context.Context) (map[string]string, error) {
 
 	ret := map[string]string{}
 	for name, signature := range state.ImageSignatures {
-		json, err := json.Marshal(signature)
-		if err != nil {
-			return nil, err
-		}
-		ret[name] = string(json)
+		ret[fmt.Sprintf("%s_KEY_ID_%s", prefix, name)] = signature.KeyID
+		ret[fmt.Sprintf("%s_%s", prefix, name)] = signature.Signature
```

Comment:
Same as above, let's guard here for empty strings:
```suggestion
		if signature.KeyID != "" {
			ret[fmt.Sprintf("%s_KEY_ID_%s", prefix, name)] = signature.KeyID
		}
		if signature.Signature != "" {
			ret[fmt.Sprintf("%s_%s", prefix, name)] = signature.Signature
		}
```